equalizer.css

```diff
@@ -19,8 +19,7 @@ a,
 button,
 select,
 textarea {
-  color: #717171;
-  display: inline-block;
+  color: #333333;
```

Comment:
#333

Reply:
CSS should have the complete hexadecimal IMO

Reply:
I think it's not necessary to have it with 3 numbers in css. This should be generated automatically by the sass builder (I'm not sure if you altered this manually or built it properly).

Reply:
In this case, the three-digit notation is acting as a shorthand. I think this should be respected following our style guidelines